tools/bump-version

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
```

Comment:
should this file have a file extension?

Reply:
According to @maxammann not :D

Reply:
Not required and it is normal to hide the language for unix-like scripts :)